packages/localizations/src/en-US.ts

```diff
@@ -325,6 +325,11 @@ export const enUS: LocalizationResource = {
         subtitleNewTab: 'Return to the newly opened tab to continue',
         titleNewTab: 'Signed in on other tab',
       },
+      clientMismatch: {
+        subtitle:
+          'Open the verification link on the device and browser from which you initiatied the sign in to continue',
```

Comment:
🔧 Suggestions welcome to improve this message!

Reply:
I find it a bit long, but i cannot think of a shorter version atm.